### PR TITLE
Clean dev

### PR DESCRIPTION
**Pull request by BlandineM**
DEBUG sur toutes les pages pour avoir une DEV clean et utilisable

### CODE REVIEW

File: frontend/src/App.js [hunk 1]
```diff
@@ -1,30 +1,22 @@
...
   return (
     <div className="App">
-      <ListCar />
       <Switch>
         <Route path="/identification" component={Identification} />
         <Route path="/login" component={Login} />
         <Route path="/signup" component={Signup} />
-        <Route path="/addCar" component={AddCar}
+        <Route path="/addCar" component={AddCar} />
+        <Route path="/" component={HomePage} />
```
Comment (laurentmahieu) [inline frontend/src/App.js:18]: exact path ;)

File: frontend/src/components/CarCard.jsx [hunk 2]
```diff
@@ -15,17 +16,25 @@ export default function CarCard() {
...
-        <h3>Kilométrage : {user.current_mileage}</h3>
+        <h3>Kilométrage : {user.current_mileage} km</h3>
         <h1>
           {user.brand} | {user.model}
         </h1>
         <h3>
           {user.motorisation} ({user.horse_power} CH)
           {user.production_year}
         </h3>
+        <div className="choix">
```
Comment (laurentmahieu) [inline frontend/src/components/CarCard.jsx]: className in english please

File: frontend/src/components/style/CarCard.scss [hunk 3]
```diff
@@ -1,24 +1,45 @@
...
   h3 {
-    color: rgb(102, 102, 102);
+    color: var(--ternaryColorF);
     font-size: 0.9em;
+    font-family: DINPro-Light;
   }
   .info {
     width: 100vw;
   }
+  .choix {
```
Comment (laurentmahieu) [inline frontend/src/components/style/CarCard.scss]: className in english please

File: frontend/src/components/ListCar.jsx [hunk 4]
```diff
@@ -1,66 +1,109 @@
...
-            >
-              Moteur
-            </button>
+            />
+            <h1>Moteur</h1>
           </div>
         </div>
 
-        <div className="module" id="freins">
+        <div className="module" id="pneus">
```
Comment (laurentmahieu) [inline frontend/src/components/ListCar.jsx]: id in english

File: frontend/src/components/ListCar.jsx [hunk 5]
```diff
@@ -1,66 +1,109 @@
...
-            <img src="/pictures/logos/Logo_MemoraCar_Def-03.png" alt="Logo" />
+            <img src="/pictures/icons/blue_car_tire.png" id="pneus" alt="/" />
             <div>
-              <h1>Freins</h1>
+              <h1>Pneus</h1>
             </div>
           </div>
         </div>
 
         <div className="module" id="chassis">
```
Comment (laurentmahieu) [inline frontend/src/components/ListCar.jsx]: id in english

File: frontend/src/components/ListCar.jsx [hunk 6]
```diff
@@ -1,66 +1,109 @@
...
+            />
+            <h1>Moteur</h1>
           </div>
         </div>
 
-        <div className="module" id="freins">
+        <div className="module" id="pneus">
           <div id="imgIcone">
-            <img src="/pictures/logos/Logo_MemoraCar_Def-03.png" alt="Logo" />
+            <img src="/pictures/icons/blue_car_tire.png" id="pneus" alt="/" />
```
Comment (laurentmahieu) [inline frontend/src/components/ListCar.jsx]: id in english

File: frontend/src/components/ListCar.jsx [hunk 7]
```diff
@@ -1,66 +1,109 @@
...
-              <h1>Freins</h1>
+              <h1>Pneus</h1>
             </div>
           </div>
         </div>
 
         <div className="module" id="chassis">
           <div id="imgIcone">
-            <h1>Chassis</h1>
+            <img src="/pictures/icons/blue_chassis.png" id="chassis" alt="/" />
```
Comment (laurentmahieu) [inline frontend/src/components/ListCar.jsx]: id in english

File: frontend/src/components/ListCar.jsx [hunk 8]
```diff
@@ -1,66 +1,109 @@
...
+            <div>
+              <h1>Chassis</h1>
+            </div>
           </div>
         </div>
 
-        <div className="module" id="pneus">
+        <div className="module" id="electricité">
```
Comment (laurentmahieu) [inline frontend/src/components/ListCar.jsx]: id in english

File: frontend/src/components/ListCar.jsx [hunk 9]
```diff
@@ -1,66 +1,109 @@
...
+          <div id="imgIcone">
+            <img
+              src="/pictures/icons/blue_electricity.png"
+              id="electricite"
```
Comment (laurentmahieu) [inline frontend/src/components/ListCar.jsx]: id in english

File: frontend/src/components/ListCar.jsx [hunk 10]
```diff
@@ -1,66 +1,109 @@
...
+              alt="/"
+            />
+
+            <h1>Electricité</h1>
+          </div>
+        </div>
+
+        <div className="module" id="echappement">
```
Comment (laurentmahieu) [inline frontend/src/components/ListCar.jsx]: id in english

File: frontend/src/components/ListCar.jsx [hunk 11]
```diff
@@ -1,66 +1,109 @@
...
           <div id="imgIcone">
-            <h1>Pneus</h1>
+            <img
+              src="/pictures/icons/red_tailpipe.png"
+              id="echappement"
```
Comment (laurentmahieu) [inline frontend/src/components/ListCar.jsx]: id in english

File: frontend/src/components/ListCar.jsx [hunk 12]
```diff
@@ -1,66 +1,109 @@
...
+              alt="Logo"
+            />
+            <div>
+              <h1>Echappement</h1>
+            </div>
           </div>
         </div>
 
-        <div className="module" id="carrosserie">
+        <div className="module" id="freins">
```
Comment (laurentmahieu) [inline frontend/src/components/ListCar.jsx]: id in english

File: frontend/src/components/ListCar.jsx [hunk 13]
```diff
@@ -1,66 +1,109 @@
...
           <div id="imgIcone">
-            <h1>Carrosserie</h1>
+            <img src="/pictures/icons/blue_brake.png" id="freins" alt="/" />
```
Comment (laurentmahieu) [inline frontend/src/components/ListCar.jsx]: id in english

File: frontend/src/components/ListCar.jsx [hunk 14]
```diff
@@ -1,66 +1,109 @@
...
+            <div>
+              <h1>Freins</h1>
+            </div>
           </div>
         </div>
 
-        <div className="module" id="electricite">
+        <div className="module" id="carosserie">
```
Comment (laurentmahieu) [inline frontend/src/components/ListCar.jsx]: id in english

File: frontend/src/components/ListCar.jsx [hunk 15]
```diff
@@ -1,66 +1,109 @@
...
           <div id="imgIcone">
-            <h1>Electricite</h1>
+            <img
+              src="/pictures/icons/blue_bodywork.png"
+              id="carrosserie"
```
Comment (laurentmahieu) [inline frontend/src/components/ListCar.jsx]: id in english

File: frontend/src/components/ListCar.jsx [hunk 16]
```diff
@@ -1,66 +1,109 @@
...
+              alt="/"
+            />
+            <div>
+              <h1>Carrosserie</h1>
+            </div>
           </div>
         </div>
 
-        <div className="module" id="echeances">
+        <div className="module" id="controle technique">
```
Comment (laurentmahieu) [inline frontend/src/components/ListCar.jsx]: id in english

File: frontend/src/components/ListCar.jsx [hunk 17]
```diff
@@ -1,66 +1,109 @@
...
           <div id="imgIcone">
-            <h1>Echeances</h1>
+            <img
+              src="/pictures/icons/orange_technical_control.png"
+              id="controletechnique"
```
Comment (laurentmahieu) [inline frontend/src/components/ListCar.jsx]: id in english

Comment (laurentmahieu) [reply frontend/src/components/ListCar.jsx]: and every id of this composent should have an unique utilization.. if not it should be a className

File: frontend/src/pages/style/AddCar.scss [hunk 18]
```diff
@@ -31,8 +32,9 @@
     flex-direction: column;
     align-items: center;
     h2 {
-      color: var(--secondColorF);
+      color: var(--ternaryColorF);
       font-size: 1em;
+      font-family: DINPro-Bold;
     }
     input {
       text-align: center;
```
Comment (laurentmahieu) [inline frontend/src/pages/style/AddCar.scss:40]: no font family for input ?

**Review by laurentmahieu — CHANGES_REQUESTED** [review]
some change you have to.
follow the comments ;)

**Review by laurentmahieu — DISMISSED** [review]

File: frontend/src/components/HomePage.jsx [hunk 19]
```diff
@@ -1,25 +1,19 @@
...
       <CarCard />
-      <button
-        onClick={() => {
-          setToggle(!toggle);
-        }}
-      >
-        {toggle ? "historic" : "ListCar"}
-      </button>
-      {toggle ? <ListCar /> : <Historic />}
+      <Switch>
```
Comment (LoicBrassart) [inline frontend/src/components/HomePage.jsx:12]: The Switch should be in App.js

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Duplicate Router, but good job aside from that ^^

**Review by laurentmahieu — APPROVED** [review]

**Review by laurentmahieu — APPROVED** [review]

**Review by LoicBrassart — APPROVED** [review]
Fine by me!

**Pull request merged**